src/service/metrics/prom.rs

```diff
@@ -417,13 +430,85 @@ pub async fn remote_write(
             "200",
             org_id,
             "",
-            StreamType::Metrics.to_string().as_str(),
+            &StreamType::Metrics.to_string(),
         ])
         .inc();
 
     Ok(HttpResponse::Ok().into())
 }
 
+pub(crate) async fn get_metadata(org_id: &str, req: RequestMetadata) -> HttpResponse {
```

Comment:
I suggest return a struct of `meta` not return a `httpResponse`, because I think the service is a common logic by HTTP or grpc or other module will call it, the `httpResponse` should package at `handler/http`

Reply:
Done.